### PR TITLE
Claudia 5376 configuration properties

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg 
Update package generator for configuration parameters
Tested with PuppetExample and ChefExample

### CODE REVIEW

File: murano-apps/ExamplePuppet/Resources/DeployExamplePuppet.template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Scripts:
+  executeRecipe:
+    Type: Puppet
+    Version: 1.0.0
+    EntryPoint: test::install
+    Files:
+        -  test/manifests/install.pp
+    Options:
+      captureStdout: true
+      captureStderr: true
```
Comment (flopezag) [inline murano-apps/ExamplePuppet/Resources/DeployExamplePuppet.template]: add new line

File: murano-apps/ExamplePuppet/Resources/scripts/test/manifests/install.pp [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  $custom_att_01=hiera('test', 'att1')
+  $custom_att_02=hiera('test', 'att2')
+
+  file {"test_puppet":
+          path => "/tmp/test_puppet",
+          ensure => present,
+          mode => 0640,
+          content => "Operation: install; Product: test-att-01; Version: ${version}; Att01: ${custom_att_01}; Att02: ${custom_att_02}"
+  }
+}
```
Comment (flopezag) [inline murano-apps/ExamplePuppet/Resources/scripts/test/manifests/install.pp]: add new line

Comment (hmunfru) [inline murano-apps/ExamplePuppet/Resources/scripts/test/manifests/install.pp]: DONE

File: murano-apps/ExamplePuppet/Resources/DeployExamplePuppet.template (hunk 1 above)
Comment (hmunfru) [inline murano-apps/ExamplePuppet/Resources/DeployExamplePuppet.template]: DONE

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**